src/es/components/molecules/buttons/Buttons.js

```diff
@@ -111,7 +111,53 @@ export default class Buttons extends Shadow() {
           button.link = button.link + '?' + filteredURLParams
         }
       }
+      if (button.event?.includes('AdvisoryText')) {
+        const parentDiv = document.createElement("div")
 
+        this.overLayButton = document.createElement("ks-a-button")
+        this.overLayButton.setAttribute("namespace", "button-primary-")
+        this.overLayButton.setAttribute("color", "secondary")
+        this.overLayButton.setAttribute("request-event-name", "dialog-open-checkout-overlay")
+        this.overLayButton.setAttribute("click-no-toggle-active", "")
+        this.overLayButton.innerHTML = button.text
+
+        this.addEventListener("dialog-open-checkout-overlay", () => {
+          new Promise(resolveCheckout => {
+            this.dispatchEvent(new CustomEvent('checkout-overlay-api', {
+              detail: {
+                resolveCheckout,
+                checkoutOverlayAPI: `https://dev.klubschule.ch${button.event}`
```

Comment:
It is better to provide the URL https://dev.klubschule.ch as attributes, e.g. checkout-overlay-url="https..."

Reply:
Ahh sry. This was only for testing reasons as the relative url we are getting does not work on local instance! This should be removed once we are merging